desktop-widgets/simplewidgets.cpp

```diff
@@ -500,109 +500,52 @@ void DiveComponentSelection::buttonClicked(QAbstractButton *button)
 	}
 }
 
-TagFilter::TagFilter(QWidget *parent) : QWidget(parent)
+FilterBase::FilterBase(FilterModelBase *model_, QWidget *parent)
+	: QWidget(parent)
+	, model(model_)
```

Comment:
we prefer the comma at the end of the previous line and the parent class on the declaration line:
```
FilterBase::FilterBase(FilterModelBase *model_, QWidget **parent) : QWidget(parent),
        model(model_)
```
and then additional initializers also add the comma in the line before.
I realize that the comma at the beginning style gives smaller diffs when you change things - I just have never seen a natural language where the comma moves to a new line in doing line breaks, so it bothers my brain.

I also am not a huge fan of ```model(model_)``` as that's too easy to confuse. How about ```model(modelParameter)```